app/Resources/public/css/print.css

```diff
@@ -119,3 +93,102 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
+    border-top: 1px solid #b5cae7;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
+}
+
+.data_table th, .data_table td {
+    line-height: 18px;
+    padding: 10px;
+}
+.data_table th {
+    padding: 8px 10px;
+    font-weight: bold;
+    background-color: #eef5fa;
+    border-bottom: 2px solid #b5cae7;
+    color: #4171b5;
+}
+.data_table th a {
+    color: #333;
+}
+.data_table_no_border {
+    border-collapse: collapse;
+    border-spacing: 0;
+    font-size: 12px;
+    margin-bottom: 15px;
+    margin-top: 8px;
+    text-align: left;
+    width: 100%;
+}
+.data_table_no_border td {
+    line-height: normal;
+    padding: 6px;
+    text-align: left;
+    vertical-align: middle;
+}
+.data_table tr.row_odd {
+    background-color: #F9F9F9;
+}
+
+.data-title .page-header{
+    color: #000;
+    font-weight: bold;
+    border-bottom: 1px solid #b5cae7;
+}
+.date-calendar .page-header{
+    color: #000;
+    font-weight: bold;
+    border-bottom: 1px solid #b5cae7;
+    border-top: 1px solid #b5cae7;
+    text-align: center;
+    background-color: #EDEDED;
+    padding-top: 10px;
+    border-radis: 10px;
+}
+
+.table_print{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #CEE0CC;
+    border-spacing: 0;
+    margin: 0 0 18px;
+    background-color: #F3FFE6;
+}
+
+.table_print td {
+    padding: 5px;
+    border-bottom: 1px solid #CEE0CC;
+    border-right: 1px solid #CEE0CC;
+    border-top: 1px solid #CEE0CC;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#666666" to be "#666" (color-hex-length)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)